src/components/totals/TotalCustomMetric/__specs__/TotalCustomMetric-spec.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+
+const React = require('react');
+const enzyme = require('enzyme');
+const jsdom = require('jsdom');
+const doc = jsdom.jsdom('<!doctype html><html><body></body></html>');
+global.document = doc;
+global.window = doc.defaultView;
```

Comment:
This global stuff should probably be extracted as it will be needed for pretty much every component test and could cause the next person to come along to trip up.

Take a look at HUI as an example of how it's done there which is reasonable. See helper files:
https://github.com/everydayhero/hui/tree/master/test/helpers